fixed iTunes (#1571)
OWASP/owasp-mstg: Document/0x06d-Testing-Data-Storage.md

#### Overview

> iOS includes auto-backup features that create copies of the data stored on the device. Backups of iOS can be made locally to the device it's connected to through iTunes (till macOS Catalina) or Finder (macOS Catalina onwards), or the cloud (via the iCloud backup feature). In both cases, the backup includes nearly all data stored on the iOS device except highly sensitive data such as Apple Pay information and Touch ID settings.

Reviewer: More active.
```suggestion
iOS includes auto-backup features that create copies of the data stored on the device. You can make iOS backups from your host computer by using iTunes (till macOS Catalina) or Finder (from macOS Catalina onwards), or via the iCloud backup feature. In both cases, the backup includes nearly all data stored on the iOS device except highly sensitive data such as Apple Pay information and Touch ID settings.
```

#### Dynamic Analysis

In order to test the backup, you obviously need to create one first. The most common way to create a backup of an iOS device is by using iTunes, which is available for Windows, Linux and of course macOS (till macOS Mojave). When creating a backup via iTunes you can always only backup the whole device and not select just a single app. Make sure that the option "Encrypt local backup" in iTunes is not set, so that the backup is stored in cleartext on your hard drive.

> On macOS Catalina onwards, iTunes is not available anymore. Managing of an iOS device, including updates, backup and restore has been moved to the Finder app. The approach remains the same, as described above.

Reviewer: ```suggestion
> iTunes is not available anymore from macOS Catalina onwards. Managing of an iOS device, including updates, backup and restore has been moved to the Finder app. The approach remains the same, as described above.
```
